### ISSUE TITLE
Script vault code to extract ssh public key does not work in Bash or Zsh

### ISSUE THREAD

**Issue by CrustyBarnacle** [mttaggart/seclab#22]
Homelab Almanac - 2-building/8-vault.html example vault code and in this script does not work in Bash/Zsh.
https://github.com/mttaggart/seclab/blob/c1742c319efd9a9c0cd75fc25bbf921de3efe967/jumpbox_setup.sh#L175

The documented command/syntax produces an error:
```
➜  Vault git:(homelab) vault kv put -mount=seclab test_kv seclab_ssh_key=$(cat ~/.ssh/id_rsa.pub)
Failed to parse K=V data: invalid key/value pair "<user>@homelab-jumpbox": format must be key=value
```

The correct syntax works (example buried in [Vault Docs for SSH key-signing](https://developer.hashicorp.com/vault/docs/secrets/ssh/signed-ssh-certificates#host-key-signing) :-/):
```
➜  Vault git:(homelab) vault kv put -mount=seclab test_kv seclab_ssh_key=@$HOME/.ssh/id_rsa.pub
=== Secret Path ===
seclab/data/test_kv

======= Metadata =======
Key                Value
---                -----
created_time       2023-09-30T05:59:33.401836121Z
custom_metadata    <nil>
deletion_time      n/a
destroyed          false
version            1
```

**Comment by mttaggart** [comment]
This script successfully created the SSH key secrets in every single one of my testing labs, but I will continue validating.

**Comment by CrustyBarnacle** [comment]
I am using the latest released [KDE Neon](https://files.kde.org/neon/images/user/20230928-0718/neon-user-20230928-0718.iso) for the Jumpbox:
```
➜  Vault git:(homelab) uname -a & lsb_release -a
[1] 8343
Linux homelab-jumpbox 6.2.0-33-generic #33~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Thu Sep  7 10:33:52 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
[1]  + 8343 done       uname -a
No LSB modules are available.
Distributor ID: Neon
Description:    KDE neon 5.27
Release:        22.04
Codename:       jammy
```

**Comment by mttaggart** [comment]
Fixed in c5a4fe75a1c56df068d4de653711fb1497ecf7dd
